CONFIGURE.md

```diff
@@ -287,6 +341,62 @@ Another option to define a state storage is to use the following properties in t
     nifi.state.storage.local.auto.persistence.interval=0 sec
 
 
+### Configuring Repositories
+
+Apache MiNiFi C++ uses three repositories similarily to Apache NiFi:
+- The Flow File Repository holds the metadata (like flow file attributes) of the flow Files.
```

Comment:
the capitalization looks weird:
```suggestion
- The Flow File Repository holds the metadata (like flow file attributes) of the flow files.
```

Reply:
Updated in 4e4434732dac8f91122fc910ea067c7c752283c6